src/Support/Tag.php

```diff
@@ -5,36 +5,55 @@
 use Illuminate\Contracts\Support\Renderable;
 use Illuminate\Contracts\View\View;
 use Illuminate\Support\Collection;
+use Illuminate\Support\HtmlString;
 
+/**
+ * A representation of a HTML tag
+ */
```

Comment:
Comments could be removed imo if they don't explain a quirk or sth.
```suggestion
```

Reply:
you're totally right